### ISSUE TITLE
Copying (Cmd+C) text should not include styling

### ISSUE THREAD

**Issue by nneubauer** [IvanMathy/Boop#118]
When copying text from boop, it should not maintain styling information. If you currently copy, the text contains Menlo Regular 15pt styling (Copy from boop, paste to pages (after pressing return once)). Maybe this is a matter of opinion (Feature Request?) but I regularly use eg. TextMate to copy and remove the styling information when copying from TextMate to another source -- I wish boop would do this. I know there is a shortcut to paste without styling, but not all apps support it (I am looking at you iMovie!!).

**Comment by IvanMathy** [comment]
Hey there! Thanks for the suggestion!

It's funny I noticed the same when testing for #95, and I have a solution prototyped out. I'll maybe try to get it in the next release!

**Comment by kentquirk** [comment]
The design of the operating system is supposed to be that the clipboard supports multiple formats, and that the application doing the paste should let the user choose between formats acceptable to the application. I think it would be a mistake to eliminate the ability to have styling just because iMovie does it wrong. 

But, to cope with this problem, it seems acceptable for boop to have an option to copy without styling, either with a checkbox somewhere or, say, Shift-Copy. Or reverse it so that Shift-Copy means "copy with styling". But please don't eliminate copy styling entirely.

**Comment by IvanMathy** [comment]
Yep, I intend to keep copy with colors as the alternative option and plain text copy as the main cmd+c shortcut. Thanks for the input!

**Comment by swinggraphics** [comment]
> Yep, I intend to keep copy with colors as the alternative option and plain text copy as the main cmd+c shortcut. Thanks for the input!

That will be great. I almost never want styled text, so having this option would save a step…that being when I forget it's styled and have to undo then re-paste as plain. :)
